xclim/core/formatting.py

```diff
@@ -247,8 +247,9 @@ def parse_doc(doc: str) -> dict[str, str]:
 
 
 def _parse_parameters(section):
-    """Parse the 'parameters' section of a docstring into a dictionary mapping the parameter name to its description and, potentially, to its set of choices.
+    """Parse the 'parameters' section of a docstring into a dictionary.
 
+    Works by mapping the parameter name to its description and, potentially, to its set of choices.
     The type annotation are not parsed, except for fixed sets of values (listed as "{'a', 'b', 'c'}").
```

Comment:
here again, it's nice I agree with in this specific case, but seems a bit powerful. It seems that this could lead to confusion. If I have documentation that is not OK with syntax, but understandable, this could change things and make things less understandable?

Reply:
That was a manual change by me to bring the function title down to a reasonable length. If this changes the meaning entirely, I can revert that